Sources_v3/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -157,6 +157,16 @@ public extension ChannelListControllerGeneric {
             self?.callback { completion?(error) }
         }
     }
+    
+    /// Marks all channels for a user as read.
+    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    func markAllRead(completion: ((Error?) -> Void)? = nil) {
+        worker.markAllRead { [weak self] error in
+            self?.callback {
+                completion?(error)
+            }
+        }
+    }
```

Comment:
missing tests + guard for new channels

Reply:
Fixed in 00fda7371027455896dc395d5e012eca690580ef
There's no nee for guard for new channels, since this endpoint will mark all channels existing at the point of calling as read

Reply:
Indeed 👍